cms/startup.py

```diff
@@ -9,6 +9,8 @@
 
 from django_startup import autostartup
 from monkey_patch import django_utils_translation
+from util import keyword_substitution
+from lms.startup import get_keyword_function_map
```

Comment:
I think the cms settings files import lms:
https://github.com/Stanford-Online/edx-platform/blob/master/cms/envs/common.py#L30

One could argue that startup.py is also such a special case instance, where importing like we do is ok.